bootstrap/fabric/src/main/java/org/geysermc/geyser/platform/fabric/world/GeyserFabricWorldManager.java

```diff
@@ -74,11 +74,8 @@ public void sendLecternData(GeyserSession session, int x, int z, List<Vector3i>
             }
 
             LevelChunk chunk = player.level().getChunk(x, z);
-            final int chunkBlockX = x << 4;
-            final int chunkBlockZ = z << 4;
             for (Vector3i vector : vectors) {
-                BlockEntity blockEntity = chunk.getBlockEntity(new BlockPos(chunkBlockX + vector.getX(),
-                        vector.getY(), chunkBlockZ + vector.getZ()));
+                BlockEntity blockEntity = chunk.getBlockEntity(new BlockPos(vector.getX(), vector.getY(), vector.getZ()));
```

Comment:
https://github.com/GeyserMC/Geyser/pull/4263/commits/df48bcab7c3f8713635a4857e220830b6000ca27
Do spigot/fabric differ here?

Reply:
I was just correcting a mistake I made initially as now the absolute position is passed instead of relative to the chunk. So the offsets here would make the position incorrect.

Reply:
I see, thanks for clearing up :)